src/series/point.js

```diff
@@ -26,16 +28,13 @@ export default function() {
             var filteredData = data.filter(base.defined);
 
             var g = dataJoin(this, filteredData);
-
             g.enter()
                 .attr('transform', containerTransform)
-                .append('circle');
+                .append('path')
+                .attr('d', symbol.size(size).type(type));
```

Comment:
As we have changed from a `circle` to a `d`, are there any affected visual tests or site examples which are accessing the circle in a decorator?